server/app/services/webhook_service.py

```
@@ -1926,46 +1931,46 @@ async def _send_google_review_sms(
             if call.service_record and call.service_record.customer_name:
                 customer_name = call.service_record.customer_name
-            
+
             # Extract first name for personalization
             customer_first_name = customer_name.split(' ')[0] if customer_name else "Customer"
-            
+
             # Get business name
             rooftop_name = rooftop.name or "us"
             organization = rooftop.organization if rooftop else None
             business_name = organization.name if organization else "Our Team"
```

Ruff (0.15.2)

[error] Local variable `business_name` is assigned to but never used

Remove assignment to unused variable `business_name`

(F841)